core/services/keeper/registry_synchronizer_core.go

```diff
@@ -59,14 +58,15 @@ type RegistrySynchronizer struct {
 // NewRegistrySynchronizer is the constructor of RegistrySynchronizer
 func NewRegistrySynchronizer(opts RegistrySynchronizerOptions) *RegistrySynchronizer {
 	mailRoom := MailRoom{
-		mbUpkeepCanceled:   utils.NewMailbox[log.Broadcast](50),
-		mbSyncRegistry:     utils.NewMailbox[log.Broadcast](1),
-		mbUpkeepPerformed:  utils.NewMailbox[log.Broadcast](300),
-		mbUpkeepRegistered: utils.NewMailbox[log.Broadcast](50),
+		mbUpkeepCanceled:    utils.NewMailbox[log.Broadcast](50),
+		mbSyncRegistry:      utils.NewMailbox[log.Broadcast](1),
+		mbUpkeepPerformed:   utils.NewMailbox[log.Broadcast](300),
+		mbUpkeepRegistered:  utils.NewMailbox[log.Broadcast](50),
+		mbUpkeepGasLimitSet: utils.NewMailbox[log.Broadcast](50),
```

Comment:
@RyanRHall 's comment: Let's increase these by a factor of 10 /100 right now.  During load testing let's try to understand the exact limits here

Reply:
created 38160 to track this